### PR TITLE
chore(new-ui): change line height of form item

### PR DESCRIPTION
**Pull request by sajclarke**
**Please make sure the following boxes are checked before submitting your Pull Request:**

- [x] I've added this PR's link to its Asana task(s)
- [ ] If this PR adds new code that is not going to change in the near future, it includes unit tests to cover it.

## This PR: 

Fixes: 
[Need to change the line height for the confirmation notification](https://app.asana.com/0/427839295975995/1115255954083335)

### CODE REVIEW

**Comment by RafaelVidaurre** [comment]
@sajclarke changes not implemented here yet

File: packages/new-polymath-ui/src/components/inputs/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ export const CheckboxPrimitive: FC<Props> = ({
           onChange={e => {
             onChange(e.target.checked);
           }}
+          style={{ border: 'red' }}
```
Comment (Grsmto) [inline packages/new-polymath-ui/src/components/inputs/Checkbox/Checkbox.tsx]: Not sure why there is a red border applied here.

File: packages/new-polymath-ui/src/components/inputs/Checkbox/styles.ts [hunk 2]
```diff
@@ -34,6 +39,7 @@ export const CheckboxInput = styled.label`
   height: 1.125rem;
   background-color: #fff;
   user-select: none;
+  vertical-align: top;
```
Comment (Grsmto) [inline packages/new-polymath-ui/src/components/inputs/Checkbox/styles.ts]: I don't think this has any effect as the parent is `flex` and vertically centered.

File: packages/new-polymath-ui/src/components/inputs/Checkbox/Checkbox.tsx [hunk 3]
```diff
@@ -39,7 +39,7 @@ export const CheckboxPrimitive: FC<Props> = ({
 
   return (
     <Fragment>
```
Comment (Grsmto) [inline packages/new-polymath-ui/src/components/inputs/Checkbox/Checkbox.tsx]: That `<Fragment>` wrapper is not needed, not sure why it was there.

File: packages/new-polymath-ui/src/components/inputs/Checkbox/Checkbox.tsx (hunk 1 above)
Comment (sajclarke) [reply packages/new-polymath-ui/src/components/inputs/Checkbox/Checkbox.tsx]: This was a mistake. Will be removed

File: packages/new-polymath-ui/src/components/inputs/Checkbox/Checkbox.tsx (hunk 3 above)
Comment (sajclarke) [reply packages/new-polymath-ui/src/components/inputs/Checkbox/Checkbox.tsx]: Will be removed

File: packages/new-polymath-ui/src/components/inputs/Checkbox/styles.ts (hunk 2 above)
Comment (sajclarke) [reply packages/new-polymath-ui/src/components/inputs/Checkbox/styles.ts]: Agreed

**Comment by RafaelVidaurre** [comment]
@sajclarke conflicts here!

**Pull request merged**